lib/asciidoctor/latex/converter.rb

```diff
@@ -89,35 +89,47 @@
 module Asciidoctor::LaTeX
   module Html5ConverterExtensions
 
-    def environment node
+    ENV_CSS = "+++<div style='line-height:1.5em;font-size:1.05em;font-style:oblique;margin-bottom:1.5em'>+++"
+    DIV_END = '+++</div>+++'
+    TABLE_ROW_END = '+++</tr></table>+++'
+
+    def info node
+
+      attrs =  node.attributes
+
+      warn "\n    node: #{node.node_name}".cyan
+      warn "   attrs: #{attrs}".cyan
+      warn "   title: #{attrs['title']}".cyan
+      warn "    role: #{attrs['role']}".cyan
+      warn "   level: #{attrs['level']}".cyan
+      warn " options: #{attrs['options']}".cyan
+      warn " type   : #{attrs['type']}".cyan
+      warn " caption: #{node.caption}".red
+      warn "      id: #{attrs['id']}".cyan
+      warn " content: #{node.content}".blue
 
-      warn "\n    node: #{node.node_name}".cyan if $VERBOSE
-      warn "   attrs: #{node.attributes}".cyan if $VERBOSE
-      warn "    role: #{node.attributes['role']}".cyan if $VERBOSE
-      warn " options: #{node.attributes['options']}".cyan if $VERBOSE
-      warn " type: #{node.attributes['type']}".cyan if $VERBOSE
-      warn "    topu: #{node.attributes['topu']}".cyan if $VERBOSE
-      warn "      id: #{node.attributes['id']}".cyan if $VERBOSE
-      warn " content: #{node.content}".blue if $VERBOSE
+    end
+
+    def environment node
 
+      info node if $VERBOSE
+      options = node.attributes['options']
+      attrs = node.attributes
 
-      if node.attributes['role'] == 'equation'
-        puts "hc: role = equation".magenta
-        node.attributes['title'] = nil
-        number_part = '<td style="text-align:right">' + "(#{node.attributes['equation_number']}) </td>"
+      if attrs['role'] == 'equation'
+        node.title = nil
+        number_part = '<td style="text-align:right">' + "(#{node.caption}) </td>"
         number_part = ["+++ #{number_part} +++"]
         equation_part = ['+++<td>+++'] + ['\\['] + node.lines + ['\\]'] + ['+++</td>+++']
-        table_style='style="width:100%; border-collapse:collapse"'
```

Comment:
@jirutka, @mojavelinux recommended that we append a stylesheet.  This would certainly be better than hardcoding the css.  Do you know how to do this?